views/src/app/Auth/pages/Home/Home.js

```diff
@@ -27,23 +28,21 @@ class Home extends Component {
 
     render() {
         return (
-            <div>
-                <Container fluid="lr">
-                    <Row>
-                        <Col className="post-container" style={{backgroundColor: "darkgray", padding: "1em"}}>
-                        <h4>Latest Posts:</h4>
-                            {this.state.posts.map(post => (
-                                <Col className="post">
-                                    <h5>Author: {post.author.username}</h5>
-                                    <h5>{post.title}</h5>
-                                    <p className="content" dangerouslySetInnerHTML={{ __html: post.content }}></p>
-                                    <hr />
+            <Container fluid="sm">
+                <Row className="posts-container">
+                    {this.state.posts.map(post => (
+                        <Row className="posts-container">
```

Comment:
you'll want to add a key here as well 🔑 
```suggestion
                        <Row key={post._id} className="posts-container">
```